### PR TITLE
add basic blog structure

### PR DESCRIPTION
**Pull request by dkuntze**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Test URLs:
- Before: https://main--shredit--stericycle.aem.page/en-us/blog/10-documents-to-take-to-communtiy-shredding-event
- After: https://related-content-blog--shredit--stericycle.aem.page/en-us/blog/10-documents-to-take-to-communtiy-shredding-event
- https://related-content-blog--stericycle-shared--aemsites.aem.page/en-us/blog/10-documents-to-take-to-communtiy-shredding-event

### CODE REVIEW

**Comment by dkuntze** [comment]
Will add the 'dynamic' article pull logic styles in #31

File: styles/styles.css [hunk 1]
```diff
@@ -326,3 +326,31 @@ main .cyan-headings {
...
 }
+
+/* blog styles */
+
+.blog-page div.related-content {
+  background-color: #f2f6f7;
+  width: 100%;
+}
+
+@media (width >= 992px) {
```
Comment (ericvangeem) [inline styles/styles.css:337]: I believe we have decided to not use this style of media query due to browser compat issues with older versions of iOS Safari - see https://github.com/aemsites/stericycle-shared/pull/33/files

Comment (seanathero) [reply styles/styles.css:337]: We added a stylelint rule to check for this as part of that PR so we should be good moving forward.

Comment (dkuntze) [reply styles/styles.css]: @seanathero fixed. I also added the stylelint fix to this branch

**Review by seanathero — APPROVED** [review]

**Pull request merged**